index.bs

```diff
@@ -115,29 +119,36 @@ Long Task timing involves the following new interfaces:
     };
 </pre>
 
-{{PerformanceLongTaskTiming}} extends the following attributes of {{PerformanceEntry}} interface:
+The values of the attributes of a {{PerformanceLongTaskTiming}} are set in the processing model in [[#report-long-tasks]]. The following provides an informative summary of how they will be set.
+
+The {{PerformanceEntry/name}} attribute's getter will return one of the following strings:
 
-* The {{PerformanceEntry/name}} attribute must return a {{DOMString}} that provides minimal frame attribution.
+: "<code><dfn>unknown</dfn></code>"
+:: The long task did not originate from an event loop <a>task</a>.
+: "<code><dfn>self</dfn></code>"
+:: The long task originated from an event loop <a>task</a> within this <a>browsing context</a>.
+: "<code><dfn>same-origin-ancestor</dfn></code>"
+:: The long task originated from an event loop <a>task</a> within a <a lt="same origin">same-origin</a> <a>ancestor browsing context</a>.
+: "<code><dfn>same-origin-descendant</dfn></code>"
+:: The long task originated from an event loop <a>task</a> within a <a lt="same origin">same-origin</a> <a lt="list of the descendant browsing contexts">descendant browsing context</a>.
+: "<code><dfn>same-origin</dfn></code>"
+:: The long task originated from an event loop <a>task</a> within a <a lt="same origin">same-origin</a> <a>browsing context</a> that is not in the same <a>unit of related browsing contexts</a>.
```

Comment:
Looking at the definition of `unit of related browsing contexts` it seems there's also opener browsing contexts, would those also be included in this case? Maybe we could define this case as same-origin but not ancestor or descendant.